src/Illuminate/Validation/Rules/DatabaseRule.php

```diff
@@ -154,7 +154,7 @@ public function whereNotNull($column)
      * Set a "where in" constraint on the query.
      *
      * @param  string  $column
-     * @param  \Illuminate\Contracts\Support\Arrayable|array  $values
+     * @param  \Illuminate\Contracts\Support\Arrayable|\BackedEnum|array|string  $value
```

Comment:
It's false!
The param is `$values` instead of `$value`

Reply:
There's also no reason for these changes, you should only be passing an array of items to `whereIn` and `whereNotIn`.


The more correct docs would be:

```php
 * @param  \Illuminate\Contracts\Support\Arrayable<mixed>|array<mixed>  $values
```